explore/src/main/scala/explore/targeteditor/ElevationPlotSection.scala

```diff
@@ -43,17 +43,17 @@ import react.semanticui.elements.button.ButtonGroup
 import java.time.ZonedDateTime
 
 final case class ElevationPlotSection(
-  uid:              User.Id,
-  tid:              Target.Id,
-  scienceMode:      Option[ScienceMode],
-  coords:           Coordinates
-)(implicit val ctx: AppContextIO)
+  uid:           User.Id,
+  tid:           Target.Id,
+  scienceMode:   Option[ScienceMode],
+  coords:        Coordinates
+)(using val ctx: AppContextIO)
```

Comment:
With the `import props.given` below, does this need a name anymore?

Reply:
Unfortunately, it does: we need it as a `val`, which forces us to declare a name.

Reply:
Ah, yeah. Second parameter list is private by default.